jormungandr-lib/src/interfaces/vote/mod.rs

```diff
@@ -90,10 +97,20 @@ impl From<VotePlanDef> for VotePlan {
             vpd.committee_end,
             vpd.proposals,
             vpd.payload_type,
+            vpd.committee_member_public_keys,
         )
     }
 }
 
+fn deserialize_committee_member_public_keys<'de, D>(
+    deserializer: D,
+) -> Result<Vec<chain_vote::MemberPublicKey>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    Ok(Vec::new())
+}
+
```

Comment:
Where do we deserialize from? Is it from some file?

Reply:
Yes, it is from a yaml/json formated file. It is used from `jcli certificate new vote-plan`.
The actual format is:
```json
{
 "payload_type": "public",
 "vote_start": {
  "epoch": 1,
  "slot_id": 0
 },
 "vote_end": {
  "epoch": 6,
  "slot_id": 0
 },
 "committee_end": {
  "epoch": 10,
  "slot_id": 0
 },
 "proposals": [
  {
   "external_id": "4f0561ca45d4b2c4cccb0d55239e0635d513ef66d7c66b0a0d4187b2ba240066",
   "options": 3,
   "action": "off_chain"
  }
 ]
}
```
right now you could add another field to that definition for the privacy committee members:
```json
 "committee": [
  "xxxxxxxxxxxxxxxxxxxxxxx",
  "yyyyyyyyyyyyyyyyyyyyyyy",
  "zzzzzzzzzzzzzzzzzzzzzzz"
 ]
```
in the future you could expand it for all committee stuff:

```json
 "committee": {
   "privacy": [
     "xxxxxxxxxxxxxxxxxxxxxxx",
     "yyyyyyyyyyyyyyyyyyyyyyy",
     "zzzzzzzzzzzzzzzzzzzzzzz"
   ],
   "threshold": 3,
   "auth": [
     "aaaaaaaaaaaaaaaaaaaaaaa",
     "bbbbbbbbbbbbbbbbbbbbbbb",
     "ccccccccccccccccccccccc"
   ]
 }
```